src/Microsoft.AspNetCore.Mvc.ViewFeatures/project.json

```diff
@@ -47,18 +47,15 @@
       "type": "build"
     },
     "Microsoft.Extensions.WebEncoders": "1.0.0-*",
-    "Newtonsoft.Json": "8.0.3",
+    "Newtonsoft.Json": "8.0.4-beta1",
```

Comment:
I think nuget usually doesn't allow a non-prerelease package to depend on a release version. Would this be problematic when Mvc RTMs? cc @emgarten